docs/pages/integrations/apache-shardingsphere-operator-charts.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: Apache Shardingsphere Operator Charts
+permalink: integrations/apache-shardingsphere-operator-charts
+type: installation
+category: integrations
+display-title: "true"
+language: en
+list: include
+image: /assets/img/integrations/apache-shardingsphere-operator-charts.svg
+---
+
+<h1>{{ page.title }} with <span style="font-weight: bold; color: green;">Meshery</span> <img src="{{ page.image }}" style="width: 35px; height: 35px;" /></h1>
+
+
+#### Category: apache-shardingsphere-operator-charts
```

Comment:
Uh-oh. This "category" is incorrect.

Reply:
This is the "model".

Reply:
@leecalcote so i need to change this category to model right 
The header should be Model